Makefile

```diff
@@ -45,7 +45,10 @@ KUBE_VERSION ?= 1.21
 KIND_CONFIG ?= kind-$(KUBE_VERSION).yaml
 
 ensure-generate-is-noop: VERSION=$(OPERATOR_VERSION)
-ensure-generate-is-noop: generate bundle
+ensure-generate-is-noop: USER=opentelemetry
```

Comment:
ci is generating the file `bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml` with below diff as USER in CI is referred to` runner`

```
--- a/bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml
+++ b/bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml
@@ -206,7 +206,7 @@ spec:
                 - --enable-leader-election
                 command:
                 - /manager
-                image: quay.io/opentelemetry/opentelemetry-operator:v0.31.0
+                image: quay.io/runner/opentelemetry-operator:v0.31.0
                 name: manager
                 ports:
                 - containerPort: 9443
Build failed: the bundle, config files has been changed but the generated bundle, config files aren't up to date. Run 'make bundle' and update your PR.
make: *** [Makefile:53: ensure-generate-is-noop] Error 1
```

So setting the USER as `opentelemetry` for the target `ensure-generate-is-noop`, would definitely appreciate any other thoughts for this fix. :)